api/src/DTO/Invitation.php

```diff
@@ -30,7 +30,7 @@
             ],
         ],
         self::ACCEPT => [
-            'security' => 'is_fully_authenticated()',
+            'security' => 'is_authenticated()',
```

Comment:
Ich würde hier noch irgendwie erwähnen (ich habe gerne commit messages, andere eher nicht), dass das "remember me" user hier auch einschliesst.
is_fully_authenticated ist anscheinend nur true, wenn man sich in der momentanen session eingeloggt hat...
(haben wir überhaupt eine session?)
https://symfonycasts.com/screencast/symfony-security/is-auth#remember-me-authed-is-authenticated-remember

Reply:
Ja, richtig. Offizielle docu dazu ist hier:
https://symfony.com/doc/current/security/expressions.html

Momentan verwenden wir die remember me Funktion nicht. Ob das mit JWT überhaupt Sinn macht, oder ob man einfach die Expiry-Dauer hochschrauben kann, weiss ich jetzt auf die Schnelle nicht.

Ich sah aber keinen Grund, spezifisch `is_fully_authenticated()` zu verwenden. Das könnte gezielt Sinn machen z.B. bei Password-Change oder E-Mail-Change, braucht dann aber noch etwas Zusatzlogik, damit der Benutzer sich vorher nochmals authentifizieren muss.

Wie/wo soll ich das genau dokumentieren?

Reply:
Ich hätte das in die commit message gepackt, da ich dort auch nachgeschaut hätte.
Aber momentan reicht es wahrscheinlich, wenn es hier beim PR als kommentar steht.
Wenn jemand darüber stoplert, kann er es ja nachdokumentieren.